spec/models/event_spec.rb

```diff
@@ -1,5 +1,73 @@
 require 'rails_helper'
 
 RSpec.describe Event, type: :model do
-  # pending "add some examples to (or delete) #{__FILE__}"
+  before do
+    Rails.application.load_seed
```

Comment:
It's time to lear how to use FactoryBot :)
And please also read:  http://www.betterspecs.org/